### ISSUE TITLE
Big Query translation on WHERE clauses for field_concept_record_completeness.sql failed

### ISSUE THREAD

**Issue by jdposada** [OHDSI/DataQualityDashboard#56]
The parameterized query on this file field_concept_record_completeness.sql
produced a query like this for one of the checks

```sql
select num_violated_rows, case when denominator.num_rows = 0 then 0 else 1.0*num_violated_rows/denominator.num_rows end  as pct_violated_rows
from
(
        select COUNT(violated_rows.violating_field) as num_violated_rows
        from
        (
                select 'VISIT_OCCURRENCE.visit_type_concept_id' as violating_field, visit_occurrence.*
                from epic_2018q3_cdm_53.visit_occurrence
                where epic_2018q3_cdm_53.visit_occurrence.visit_type_concept_id = 0
        ) violated_rows
) violated_row_count,
(
        select COUNT(*) as num_rows
        from epic_2018q3_cdm_53.visit_occurrence
) denominator
;
```

The WHERE clause is this statement is invalid for Big Query. I wonder if it is necessary for other databases to reference the `@cdmDatabaseSchema` on this clause. If not, I propose we could just remove it. What do you think @alondhe? Thanks

**Comment by clairblacketer** [comment]
Can you elaborate on why the where clause is invalid? We need the where clause to evaluate this check because the 0 value indicates that the field has no value. This may be more of a sql render issue than a data quality issue.

**Comment by jdposada** [comment]
Hi, @clairblacketer thank you for your fast reply. the WHERE clause is referencing the database schema. For BQ this is the database name (e.g. epic_2018q3_cdm_53). This I believe it is not necessary for the WHERE clause to properly execute.

**Comment by clairblacketer** [comment]
Ah I see. I looked at the other WHERE clauses and I agree with you, we can remove the cdmDatabaseSchema parameter and just keep the cdmFieldName and cdmTableName.

**Comment by jdposada** [comment]
There are other files with the same issue I just found out:

- field_source_value_completeness.sql
- field_measure_value_completeness.sql

Thanks

**Comment by alondhe** [comment]
Addressed in #79, fixes forthcoming
